src/test/java/io/cdap/plugin/batch/source/ftp/FTPBatchSourceTest.java

```diff
@@ -38,30 +35,19 @@
 import java.util.Map;
 
 /**
- * Unit test for {@link FTPBatchSource.FTPBatchSourceConfig} class.
+ * Unit test for {@link FTPBatchSource} class.
  */
-
 public class FTPBatchSourceTest {
-
-  private static final String USER = "user";
-  private static final String PASSWORD_WITH_SPECIAL_CHARACTERS = "we:/_x^Yz @123#456!";
-  private static final String PASSWORD_WITHOUT_SPECIAL_CHARACTERS = "wexYz123456";
+  private static final String USER_WITH_SPECIAL_CHARACTERS = "user/12^!@_!";
```

Comment:
can we add a colon to username?

Reply:
Added, also one ETL test case is added with username and password having colon